.github/workflows/example-ios-build-check.yml

```diff
@@ -1,10 +1,10 @@
-name: Example iOS build check
+name: Example app, iOS build check (paper and fabric)
```

Comment:
Please revert this change, I made the workflow names match their filename for convenience.